src/classes/OPP_AutomatedSoftCreditsService.cls

```diff
@@ -177,22 +177,27 @@ public class OPP_AutomatedSoftCreditsService {
             return accountToAffiliations;
         }
 
+        List<String> relatedContactRoles = new List<String>();
+        Schema.DescribeFieldResult fieldResult = npe5__Affiliation__c.Related_Opportunity_Contact_Role__c.getDescribe();
+        List<Schema.PicklistEntry> picklistValues = fieldResult.getPicklistValues();
+        for (Schema.PicklistEntry picklistValue : picklistValues) {
+            relatedContactRoles.add(picklistValue.getLabel());
```

Comment:
This should use `getValue()` instead of `getLabel()`? Picklist values are what gets stored in the db, versus the label. They're normally identical, but they don't have to be.